### DISCUSSION TITLE
Fooocus and Stable Diffusion

### DISCUSSION THREAD

**Discussion by Sanafaham** [lllyasviel/Fooocus#1862] [General]
Hello all, I'm new here, how do i use Fooocus with Stable Diffusion for image generating ? Can anyone tell me please? I downloaded it and got to here . Thank you

**Comment by eddyizm** [comment]
what os are you on? Did you follow the install instructions [here](https://github.com/lllyasviel/Fooocus#download)?

**Comment by Sanafaham** [comment]
yes , it gets downloaded but wont open. Does it work for MacBook Air? Thank you so much

**Comment by eddyizm** [comment]
Does the air have an m1 or m2 chip?
_
You can install Fooocus on Apple Mac silicon (M1 or M2) with macOS 'Catalina' or a newer version. Fooocus runs on Apple silicon computers via PyTorch MPS device acceleration_

If not then I don't think it will run,
If it does tell me exactly what step in the Mac install you get to and what errors you get as well as the output of the log.
